package/ci/appveyor-desktop.bat

```diff
@@ -69,6 +69,7 @@ cmake .. ^
     -DCMAKE_INSTALL_PREFIX=%APPVEYOR_BUILD_FOLDER%/deps ^
     -DCMAKE_PREFIX_PATH=%APPVEYOR_BUILD_FOLDER%/openal;%APPVEYOR_BUILD_FOLDER%/devil;C:/Tools/vcpkg/installed/x64-windows ^
     -DWITH_ASSIMPIMPORTER=OFF ^
+    -DWITH_BASISIMPORTER=OFF -DBASISU_DIR=%APPVEYOR_BUILD_FOLDER%/basis_universal ^
```

Comment:
Adding a note here so we don't forget to flip this on.

Reply:
Oh no, I thought I did 😅

Reply:
Ah, yes, but the commit was later